From eed0658fd6f7153832c31539af54dd961e5324d9 Mon Sep 17 00:00:00 2001
From: "Raphael S. Carvalho" <raphaelsc@cloudius-systems.com>
Date: Thu, 20 Mar 2014 05:53:28 +0200
Subject: [PATCH] Add fs_mark into the OSv build process

Signed-off-by: Raphael S. Carvalho <raphaelsc@cloudius-systems.com>
---
 bootfs.manifest.skel | 1 +
 build.mk             | 3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/bootfs.manifest.skel b/bootfs.manifest.skel
index 9816e92..53cf559 100644
--- a/bootfs.manifest.skel
+++ b/bootfs.manifest.skel
@@ -5,3 +5,4 @@
 /zfs.so: zfs.so
 /tools/mkfs.so: tools/mkfs/mkfs.so
 /tools/cpiod.so: tools/cpiod/cpiod.so
+/fs_mark.so: fs_mark.so
diff --git a/build.mk b/build.mk
index aa59a8d..d7e389e 100644
--- a/build.mk
+++ b/build.mk
@@ -295,6 +295,7 @@ include $(src)/bsd/cddl/contrib/opensolaris/lib/libuutil/common/build.mk
 include $(src)/bsd/cddl/contrib/opensolaris/lib/libzfs/common/build.mk
 include $(src)/bsd/cddl/contrib/opensolaris/cmd/zpool/build.mk
 include $(src)/bsd/cddl/contrib/opensolaris/cmd/zfs/build.mk
+include $(src)/fsmark-osv/build.mk

 bsd  = bsd/net.o
 bsd += bsd/$(arch)/machine/in_cksum.o
@@ -736,7 +737,7 @@ $(jni): INCLUDES += -I /usr/lib/jvm/java/include -I /usr/lib/jvm/java/include/li

 bootfs.bin: scripts/mkbootfs.py $(java-targets) $(out)/bootfs.manifest $(tests) $(java_tests) $(tools) \
                tests/testrunner.so \
-               zpool.so zfs.so
+               zpool.so zfs.so fs_mark.so
        $(call quiet, $(src)/scripts/mkbootfs.py -o $@ -d $@.d -m $(out)/bootfs.manifest \
                -D jdkbase=$(jdkbase) -D gccbase=$(gccbase) -D \
                glibcbase=$(glibcbase) -D miscbase=$(miscbase), MKBOOTFS $@)
--
1.8.5.3
